examples/reference/panes/VTK.ipynb

```diff
@@ -26,9 +26,12 @@
     "  - w: set representation of all actors to *wireframe*\n",
     "  - v: set representation of all actors to *vertex*\n",
     "  - r: center the actors and move the camera so that all actors are visible\n",
+    "  \n",
+    "  The mouse must be over the pane to work\n",
     "  <br>**Warning**: These keybindings may not work as expected in a notebook context, if they interact with already bound keys\n",
+    "* **``orientation_widget``** (bool): A boolean to activate/deactivate the orientation widget in the 3D pane. This widget is clickable and allows to rotate the scene in one of the orthographic projection\n",
     "* **``object``** (str or object): Can be a string pointing to a local or remote file with a `.vtkjs` extension, or a `vtkRenderWindow` object  \n",
-    "\n",
+    "* **``serialize_policy``** (str): String selected between `'instantiation'` and `'display'`. It defines when the serialization of the VTK panel occurs. If `display` (default) is selected, the object is serialized only when the panel is displayed to the screen else if `instantiation` is activated, the serialization happened when the panel is created. This parameter is constant, once set it can't be modified\n",
```

Comment:
```suggestion
    "* **``serialize_policy``** (str): String selected between `'instantiation'` and `'display'`. It defines when the serialization of the VTK panel occurs. If `display` (default) is selected, the object is serialized only when the panel is displayed to the screen else if `instantiation` is activated, the serialization happened when the panel is created. This parameter is constant, once set it can't be modified\n",
```

I can't imagine any other policies can you? If there's only ever going to be two options I'd make this a boolean and call it ``serialize_on_instantiation``

Reply:
yes indeed I'll change this